src/navigation/NavigationTypes.ts

```diff
@@ -42,10 +42,14 @@ export type RoutesParams = {
   DictionaryTab: undefined
   UserVocabularyTab: undefined
   Home: undefined
-  UserVocabularyOverview: undefined
-  UserVocabularyEdit: {
-    title: string
+  OverviewUserVocabulary: undefined
+  EditUserVocabulary: {
+    headerBackLabel: string
   }
+  UserVocabularyList: {
+    headerBackLabel: string
+  }
+  UserVocabularyDetail: { document: Document }
```

Comment:
Naming is now really inconsistent, why did you rename to `OverviewUserVocabulary` and `EditUserVocabulary`? I think we should stick to the previous naming, i.e. `UserVocabularyOverview` and so on.

And what is the difference between overiew and list?

Reply:
well then lets talk about naming conventions. i think we had a discussion in this pr.. and `EditUserVocabulary` was the decision so i also changed the Overview Screen name

Reply:
Update: i talked to @SteffiStuffel  and we decided to revert this due to some misunderstandings